packages/web/components/JournalyEditor/Toolbar/Toolbar.tsx

```diff
@@ -29,176 +29,209 @@ import ToggleMarkButton from './ToggleMarkButton'
+            </ToolbarButton>
+          </div>
+          <div className="toolbar-row">
+            <ToolbarButton type="block" format="heading-two">
+              <FormatTitleIcon title="Apply heading" titleId="toolbar-title-icon" />
+            </ToolbarButton>
+            <ToolbarButton type="block" format="block-quote">
+              <FormatQuoteIcon title="Block quote" titleId="toolbar-quote-icon" />
+            </ToolbarButton>
+            <ToolbarButton type="block" format="numbered-list">
+              <FormatListNumberedIcon title="Numbered list" titleId="toolbar-list-numbered-icon" />
+            </ToolbarButton>
+            <ToolbarButton type="block" format="bulleted-list">
+              <FormatListBulletedIcon title="Bulleted list" titleId="toolbar-list-bulleted-icon" />
+            </ToolbarButton>
+
+            {isTableActivated ? (
+              <Popup
+                trigger={<span>{tableIcon}</span>}
+                position="bottom center"
+                on={['hover', 'focus']}
+                closeOnDocumentClick
+                className="editor-toolbar-popover"
+              >
+                <ToolbarTable
+                  {...options}
+                  className="editor-toolbar-popover-item"
+                  transform={addRow}
+                  icon="Add row"
+                />
+                <ToolbarTable
+                  {...options}
+                  className="editor-toolbar-popover-item"
+                  transform={deleteRow}
+                  icon="Delete row"
+                />
+                <ToolbarTable
+                  {...options}
+                  className="editor-toolbar-popover-item"
+                  transform={addColumn}
+                  icon="Add column"
+                />
+                <ToolbarTable
+                  {...options}
+                  className="editor-toolbar-popover-item"
+                  transform={deleteColumn}
+                  icon="Delete column"
+                />
+                <ToolbarTable
+                  {...options}
+                  className="editor-toolbar-popover-item"
+                  transform={deleteTable}
+                  icon="Delete table"
+                />
+              </Popup>
+            ) : (
+              tableIcon
+            )}
+          </div>
+          <div className="typewriter-sounds toolbar-row">
+            <div className="typewriter-sounds-switch-container">
+              <span>{t('typewriterSounds')}</span>
+              <SwitchToggle
+                isToggled={shouldPlayTypewriterSounds}
+                onToggle={onToggleShouldPlayTypewriterSounds}
+              />
+            </div>
+          </div>
+        </div>
+        <style jsx>{`
+          .editor-toolbar-container {
+            position: sticky;
+            // Update this value when visualViewport (diff between viewport & visualViewport)
+            // listen to changes to size of visualViewport
```

Comment:
Think this was two different comments that never quite got finished. I'd suggest this:

```suggestion
            // Push toolbar down from the top of the logical viewport to the degree that the logical viewport is outside the visual viewport
```

but IDK if that's much clearer to anyone else